### PR TITLE
chore: [Popper] remove process.env

### PR DESCRIPTION
**Pull request by benmccann**
an updated version of https://github.com/floating-ui/floating-ui/pull/2252

The failing CI checks are present on the `v2.x` branch without this change and are unrelated to the change. To see that you can look at the last PR merged to the branch and see that it had the same checks fail: https://github.com/floating-ui/floating-ui/pull/2229

### CODE REVIEW

**Comment by benmccann** [comment]
@atomiks would you be able to take a look at this one? Thanks!

**Comment by atomiks** [comment]
@FezVrasta can you take a look and do a release?

File: src/createPopper.js [hunk 1]
```diff
@@ -215,12 +149,9 @@ export function popperGenerator(generatorOptions: PopperGeneratorArgs = {}) {
...
         for (let index = 0; index < state.orderedModifiers.length; index++) {
-          if (__DEV__) {
-            __debug_loops__ += 1;
-            if (__debug_loops__ > 100) {
-              console.error(INFINITE_LOOP_ERROR);
-              break;
-            }
+          __debug_loops__ += 1;
+          if (__debug_loops__ > 100) {
+            break;
```
Comment (FezVrasta) [inline src/createPopper.js:154]: This should go away completely

**Comment by FezVrasta** [comment]
Done, released to npm

**Comment by benmccann** [comment]
a massive thank you for this!

**Pull request merged**